webservice/app/Http/Controllers/CategoriesController.php

```diff
@@ -42,11 +42,9 @@ public function fullList()
      */
     public function store(CategoryRequest $request)
     {
-        Category::create($request->only('name'));
+        $category = Category::create($request->all());
```

Comment:
I usually avoid all() because of security concerns. What do you guys have to say? @flyingluscas @vinicius73